### ISSUE TITLE
Distribution through NPM

### ISSUE THREAD

**Issue by marstonstudio** [danielstern/ngAudio#96]
Any chance of distributing this package through NPM repository? I see that bower is the recommended installation, but would be nice to also have it available through http://www.npmjs.com

**Comment by dinu88** [comment]
+1 for npm repository.

**Comment by danielstern** [comment]
Hey @marstonstudio 

Unfortunately, I'm a little busy with some other repos right now, so I recommend you investigate this and make a pull request.

If someone could make the necessary updates to `package.json`, I'd be happy to officially push the repo to NPM.

Cheers,
DS

**Comment by marstonstudio** [comment]
Got it. This would be my first foray into publishing to NPM, but I'm willing to take a look and put together a pull request.

**Comment by EpicOkapi** [comment]
Didn't seem like he was still going to do it, so I threw something together. Check it out

**Comment by santiq** [comment]
What happened with this ?

**Comment by danielstern** [comment]
As far as I can tell, this is all set?

**Comment by santiq** [comment]
Yes, and it's very quick and easy, just follow this few steps [Publishing to npm](https://gist.github.com/coolaj86/1318304)

**Comment by danielstern** [comment]
@santiq I've added you as a collaborator, please let me know if you need any other elevated permissions to resolve this

**Comment by kmccullough** [comment]
IDK if I should raise a separate issue, but the current npm package for this doesn't seem to match the package.json in this repository, and has gulp and browser-sync as a regular (non-dev) dependencies.

**Comment by danielstern** [comment]
Hmm... I'm stumped. Anyone think they can handle this one?
